src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,58 @@
+.clock {
+  @include rectangle ($clock-size, $clock-size);
+  background-color: gold;
+  border-radius: 40%;
+
+  position: relative;
+
+  &__round {
+    top: 50%;
+
+    @include rectangle ($round-clock, $round-clock);
+    @extend %watch-hands;
+    background-color: orangered;
+    z-index: 1;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+
+  }
+
+  &__seconds {
+    top: $clock-paddings;
+
+    background-color: orangered;
+    border-radius: 6px;
+
+    position: relative;
+
+    @include rectangle (10px, $seconds-hand);
+    @extend %watch-hands;
+    transform: translateX(-50%);
+    transform-origin: bottom right;
+    animation: move 60s linear infinite;
```

Comment:
it could be used 'animation-timing-function: step()' for ticking effect, but it's just a suggestion :)

Reply:
add steps for minutes watch-hand